### PR TITLE
Restaurant Page

### PR DESCRIPTION
**Pull request by jcy2704**
### Full Restaurant 

A simple and minimalistic restaurant page with a single HTML file. Used JavaScript to render the different content that was typically in a restaurant page; the menu and the contact information. To be able to render and change the different content we used DOM to make the association with the HTML file.

**The features include**:
- **The Home**, simple logo and a attention grabbing slogan with two buttons that renders the menu and contact info.
- **The Menu**, a carousel with 4 dishes, 1 for each respective meal of the day.
- **The Contact** information, where people can see the number and address.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Built With
+
+- HTML
+- CSS
+- Bootstrap
+- JavaScript
+
+## Live Preview
+
+[Book Library](https://rawcdn.githack.com/jcy2704/restaurant-page/e0fbc1481aae6d0dfd28682a1cab4748d850a68a/dist/index.html)
```
Comment (bellom) [inline README.md]: Hello @jcy2704, you need to name this `Restaurant` not `Book Library` right? 

kindly correct this 👍

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    </div>
+    <a class="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-bs-slide="prev">
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Previous</span>
+    </a>
+    <a class="carousel-control-next" href="#carouselExampleCaptions" role="button" data-bs-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="visually-hidden">Next</span>
+    </a>
+  </div>
```
Comment (bellom) [inline dist/index.html]: Well done @jcy2704 👍 

According to the instruction on `Odin` page, we should only have `<div id="content"></div>` inside the `body` tag. 

image

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "main": "index.js",
+  "dependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2"
+  },
+  "devDependencies": {
+    "webpack": "^5.11.1",
+    "webpack-cli": "^4.3.1"
```
Comment (bellom) [inline package.json]: We should have all necessarily used packages that should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) **in our case we are missing `webpack-dev-server`**
Kindly add `webpack-dev-server`.

We also need inform users about these script in the README file. This information can be placed in the Usage section. This way users can know the command to run to start your application.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (bellom) [inline package.json]: We need to add some scripts.

kindly double-check on this 👇 
image

We should be able to run common commands like `build`, `development` should be in `package.json` inside \`scripts'.  [Pathwright](https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635163/) in our own case here, we are missing it so I can't run `npm run build ` to generate `dist folder`, I should be able to run `npm run build`.
kindly add these two scripts `build` and `development`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @jcy2704  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @jcy2704, great job fixing all the comments outlined by the previous TSE :fireworks: CONGRATULATIONS. I hereby approve the project :+1: 

[Optional]
- Kindly add webpack-dev-server to the scripts so we can run the server with the command `npm run development` since we have it in our devDependencies

Kindly go ahead and merge into development :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**